webview/src/shared/components/dragDrop/Draggable.tsx

```diff
@@ -11,29 +10,29 @@ export interface DraggableProps {
   group: string
   disabled: boolean
   children: JSX.Element
-  dropTarget: JSX.Element
   onDrop: DragFunction
   onDragStart: DragFunction
   onDragEnter: DragFunction
   onDragEnd: DragFunction
+  onDragLeave: DragFunction
 }
 
 export const Draggable: React.FC<DraggableProps> = ({
   id,
   group,
   children,
   disabled,
-  dropTarget,
   onDrop,
   onDragEnter,
   onDragStart,
-  onDragEnd
+  onDragEnd,
+  onDragLeave
 }) => {
   const groupState = useSelector(
     (state: ExperimentsState) => state.dragAndDrop.groups[group] || {}
   )
   const dispatch = useDispatch()
-  const { draggedOverId, draggedId } = groupState
```

Comment:
Do you think it's possible to also remove it from `dragDropSlice`? And even simplify the whole thing?

Reply:
Yes! Just didn't want to make it even bigger and even more unfocused. I'll try to do this as a followup. We can keep simplifying it.